src/tests.rs

```diff
@@ -273,7 +272,6 @@ mod cli {
         let db = open_db();
 
         db.create_table_if_missing();
-        db.clear_all_tasks();
```

Comment:
The same issue as before, the test checks if an empty database exports to a empty output - your change obstructs this

Reply:
Makes sense